au4/src/projectscene/view/clipsview/timelineruler.h

```diff
@@ -0,0 +1,73 @@
+#pragma once
+
+#include <QtGui/QColor>
+#include <QtGui/QFont>
+#include <QtQuick/QQuickPaintedItem>
+
+#include "rulerformat.h"
+#include "timeformat.h"
+
+class QPainter;
+
+using TickLabelInfo = std::tuple<int, QString, RulerFormat::TickType>;
```

Comment:
It’s better to use a structure with named names, it’s much more convenient and clearer than `std::get<0>`